lib/core/widgets/home_page_app_bar.dart

```diff
@@ -152,7 +153,7 @@ class Action extends StatelessWidget {
       Widget getSuitableWidget() {
         if (isSearchEnabled) {
           return Padding(
-            key: const ValueKey("close icon"),
```

Comment:
No need to change this text as its for `ValueKey`. Its only for identifying widgets, this text won't be visible to users